yaksh/views.py

```diff
@@ -1464,7 +1464,16 @@ def show_all_questions(request):
     user_tags = questions.values_list('tags', flat=True).distinct()
     all_tags = Tag.objects.filter(id__in=user_tags)
     upload_form = UploadFileForm()
+    paginator = Paginator(questions, 10)
```

Comment:
This throws a warning if the queryset is not ordered. Use either `order_by("id")` or `order_by("-id")` for ascending and descending order respectively. Also use `get_queryset()` before filter. i.e. `Questions.objects.get_queryset().filter(<condition>)`.